terraform/wallarm-rules/main.tf

```diff
@@ -0,0 +1,252 @@
+# 
+# Define provider parameters
+# 
+provider "wallarm" {
+  api_uuid = "${var.wallarm_api_uuid}"
+  api_secret = "${var.wallarm_api_secret}"
+  api_host = "${var.api_host}"
+}
+
+# 
+# Virtual Patch section
+# 
+resource "wallarm_rule_vpatch" "splunk" {
+  attack_type =  ["sqli", "nosqli"]
+
+  action {
+    type = "iequal"
+    value = "splunk.wallarm-demo.com:88"
```

Comment:
Please take the "wallarm-demo.com" domain name from TF variable in this and several other places in the PR.